<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartAlarm</class>
 <widget class="QWidget" name="SmartAlarm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartAlarm</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTabWidget::tab-bar {
    alignment: center;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Alarm</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Timer</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="addAlarm">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
